### ISSUE TITLE
tensorflow-addons-0.6.0

### ISSUE THREAD

**Issue by long123524** [Akhilesh64/ResUnet-a#3]
How to use pip to install tensorflow-addons-0.6.0? I find not it. Thank you

**Comment by Akhilesh64** [comment]
Check if you satisfy the requirements for tensorflow-addons here: [link](https://github.com/tensorflow/addons). tensorflow-addons-0.6.0 supports python versions 3.5-3.7.

**Comment by long123524** [comment]
Thank you reply! My  python versions is 3.6, and tensorflow-gpu==2.0.0, but use "pip" failed to install tensorflow-addons==0.6.0.
I want to know how you installed it because the pip instruction can't find the tensorflow-addons==0.6.0 version.

**Comment by Akhilesh64** [comment]
I was able to install the dependencies and run the code on google colab which has default python3.7. Can you upgrade your python version to 3.7?

**Comment by long123524** [comment]
![image](https://user-images.githubusercontent.com/73996375/161433143-8b9e2d88-fa97-4d69-9b95-97647e70241d.png)
Thank you reply! I upgrade my python version to 3.7, but it still failed to install tensorflow-addons==0.6.0. Because the version is not be found.  Can you install the tensorflow-addons==0.6.0?

**Comment by Akhilesh64** [comment]
The issue is your OS. tensorflow-addons got windows support after v0.7.0. So there's no support for v0.6.0 on windows. The same code works smoothly on linux machines. What you can try is firstly install tensorflow-gpu==2.0.0 and then simply run pip install tensorflow-addons==0.7.0 to see if it is works. I've not tested this so I am not sure if this would work. See this [link](https://github.com/tensorflow/addons/issues/821) for more info.

**Comment by long123524** [comment]
Thanks four your advice. To use tensorflow-addons, we downloaded and installed higher versions of tensorlow and cuda, corresponding to tensorlow--gpu==2.5.0 and cuda==11.2. We successfully run the code, but the GPU utilization is very low, causing a slow training speed. Do you have any good suggestions to improve the problem? Thank you!
![image](https://user-images.githubusercontent.com/73996375/162953031-8d92a9b0-b5e7-4b77-976f-b0fa327654e2.png)

**Comment by Akhilesh64** [comment]
Hi @long123524 I encountered this problem too and this is due to the dataloader that we're using. If your whole dataset can fit into memory I'd recommend using model.fit() directly instead of loading data through a dataloader. Another thing you can try is apply the preprocessing to the training data([given here](https://github.com/Akhilesh64/ResUnet-a/blob/2b7bb6d9924b2fad6209988d5eb25ecacda30874/batch_preprocess.py#L25)) beforehand and save that data to disk to bypass on the fly preprocessing which is slow. Both methods will require significant code changes.

**Comment by long123524** [comment]
Thank you! I wonder what your results are like? I got a really bad prediction

**Comment by long123524** [comment]
Thank you! I wonder what your results are like? I got a really bad prediction, especially for fields mask prediction.

**Comment by Akhilesh64** [comment]
The results I got for plot boundary detection were quite good.

**Comment by long123524** [comment]
Thank your reply. I obtained the plot boundary and the plot were unsatisfactory. I want to know the numbers of your training samples, because I only use 2528 training samples with 256x256 pixels. Moreover, I want to know the extraction of your plot, it looks very better? Thank you!

**Comment by Akhilesh64** [comment]
I don't remember fully but I think I used around 10k training samples. The extraction of the plot can be done using the predict script.
